src/client/client.ts

```diff
@@ -186,7 +186,7 @@ export class GraphQLTransportWSClient {
 
     const reqId = this.generateRequestId();
     this.requests[reqId] = { options, handler };
-    this.sendMessage(this.buildMessageForRequest(reqId, MessageTypes.GQL_START, options));
+    this.sendMessage(GraphQLTransportWSClient.buildMessage(reqId, MessageTypes.GQL_START, options));
```

Comment:
maybe sendMessage shoud just get "type, payload, reqId" instead of calling buildMessage all the time?